tests/test_common_options.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+
+from utils import constants
+
+
+# Polarion TC MTA-372
+def test_list_targets():
+    kantra_path = os.getenv(constants.KANTRA_CLI_PATH)
+    command = kantra_path + ' analyze --list-targets'
+
+    output = subprocess.run(command, shell=True, check=True, stdout=subprocess.PIPE, encoding='utf-8').stdout
+
+    for i in ['hibernate-search5', 'eap', 'azure-aks', 'openjdk11', 'java-ee7', 'quarkus']:
```

Comment:
hibernate-search5 is a target ?

azure-aks should be not present in downstream.
So instead of that we can assert for azure-appservice present .

Let's also assert for some new targets .
openjdk21, jws6 n camel 4

Reply:
Hi @sshveta,

* I'll change `azure-aks` to `azure-appservice`.
* `hibernate-search5` is a valid target according to the `help` command.
* `openjdk21`, `jws6` and  `camel4` are not valid targets according to the `help` command


Here are the lists of valid sources and targets availbable in kantra right now:

Sources:

```
camel2
drools5
eap
eap4
eap5
eap6
eap7
eap7.0
eap7.1
eapxp
glassfish
hibernate-search4
hibernate3.9
hibernate4
hibernate5.0
hibernate5.1
jakarta-ee7
java
java-ee
javaee
jbpm5
jrun
log4j
openjdk11
openjdk8
oraclejdk7
orion
resin
resteasy2
rmi
rpc
seam2
soa
soa-p5
sonic
sonicesb
springboot
thorntail
weblogic
websphere
```

Targets:

```
azure-aks
azure-appservice
azure-container-apps
azure-spring-apps
camel2
camel3
cloud-readiness
drools6
eap
eap6
eap7
eap8
eapxp2
eapxp3
fsw6
fuse6
hibernate-search5
hibernate4
hibernate5
hibernate5.1
hibernate5.3
hibernate6
hibernate6.1
hibernate6.2
jakarta-ee8
jakarta-ee9
java-ee6
java-ee7
jbpm6
linux
openjdk11
openjdk17
openjdk7
openliberty
quarkus
resteasy3
rhr
```

Reply:
Hi @abrugaro @sshveta Checked on MTR we have three targets `hibernate, hibernate-search and hibernated-search5`
![Screenshot from 2023-09-18 13-31-08](https://github.com/konveyor-ecosystem/kantra-cli-tests/assets/87791967/544eb74d-a90e-44a2-8abe-b3429c05e18f)
And Tackle has `hibernate and hibernate-search`
![Screenshot from 2023-09-18 13-31-25](https://github.com/konveyor-ecosystem/kantra-cli-tests/assets/87791967/1ad829ae-a97f-408e-8b5a-07751e61e4d2)

Reply:
`hibernate-search5` is not present

Reply:
But it is present in kantra... should I remove the assertion @sshveta ?